modules/installing-oci-about-assisted-installer.adoc

```diff
@@ -0,0 +1,24 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_oci/installing-oci-assisted-installer.adoc [Using the Assisted Installer to install a cluster on OCI]
+
+:_mod-docs-content-type: CONCEPT
+[id="installing-oci-about-assisted-installer_{context}"]
+= The Assisted Installer and OCI overview
+
+From {product-title} {product-version} and later versions, you can install an {product-title} cluster on {oci-first} virtual machines (VMs) by using the Assisted Installer. You can run cluster workloads on infrastructure that supports dedicated, hybrid, public, and multiple cloud environments. Both Red Hat and Oracle test, validate, and support running {oci} in an {product-title} cluster on {oci}.
+
+{oci} provides services that can meet your needs for regulatory compliance and performance and cost-effectiveness.
```

Comment:
Alternative suggestions:
{oci} provides services that can meet your needs for regulatory compliance and for performance and cost-effectiveness. OR
{oci} provides services that can meet your needs for regulatory compliance, performance, and cost-effectiveness.